packages/smooth_app/lib/cards/product_cards/smooth_product_card_not_found.dart

```diff
@@ -68,14 +68,15 @@ class SmoothProductCardNotFound extends StatelessWidget {
             icon: Icons.add,
             padding: const EdgeInsets.symmetric(vertical: LARGE_SPACE),
             onPressed: () async {
-              await Navigator.push<void>(
-                context,
-                MaterialPageRoute<void>(
-                  builder: (BuildContext context) =>
-                      AddNewProductPage.fromBarcode(barcode),
-                ),
+              await AppNavigator.of(context).push(
+                AppRoutes.PRODUCT_CREATOR(barcode),
               );
-
+              // await Navigator.push<void>(
```

Comment:
Why do you comment this code?

Reply:
It was the previous code that opened the AddProductInformation page with the Tabs Below it.
So I Commented it and added new code Above it just to open that page from Parent Navigator.
Should I Remove that commented code?